lib/ReactViews/Custom/Chart/MomentPointsChart.jsx

```diff
@@ -62,26 +63,42 @@ class MomentPointsChart extends React.Component {
     const { id, chartItem, scales } = this.props;
     const baseKey = `moment-point-${chartItem.categoryName}-${chartItem.name}`;
     const fillColor = chartItem.getColor();
+    const isClickable = chartItem.onClick !== undefined;
+    const clickProps = point => {
+      if (isClickable) {
+        return {
+          pointerEvents: "all",
+          cursor: "pointer",
+          onClick: () => chartItem.onClick(point)
+        };
```

Comment:
Populate these props only if the point is clickable (i.e `onClick` handler is set)

Reply:
why not just go with chartItem?.onClick(point)?

Reply:
I re-wrote it this way because there are a bunch of props I want to set depending on whether the item is clickable or not.